skimage/measure/fit.py

```diff
@@ -782,15 +785,18 @@ def ransac(data, model_class, min_samples, residual_threshold,
     best_inliers = None
 
     random_state = check_random_state(random_state)
-
+    if isinstance(min_samples, float):
```

Comment:
I'm not a big fan of this change... It could be a breaking change for some users. From my reading of the code, there was nothing stopping someone from using 17.5 as `min_samples` before? And it could be convenient for some users. If we want to use some magic here, I would suggest that `min_samples` in the range (0, 1) is considered a fraction, and min_samples greater than or equal to 1 is considered an absolute value. Does that make sense?